broker-core/src/main/java/io/zeebe/broker/subscription/message/processor/ResetMessageCorrelationProcessor.java

```diff
@@ -62,20 +70,30 @@ public void processRecord(
     }
     messageState.removeMessageCorrelation(messageKey, workflowInstanceKey);
 
-    final MessageSubscription subscription =
```

Comment:
We should write an "ACKNOWLEDGE" event for the command (with intent "REJECTED"). Otherwise, we would have no follow-up event if no subscription exists.